### PR TITLE
Add new gender option of "Other"

### PR DESCRIPTION
**Pull request by Nick-Hall**

### CODE REVIEW

File: gramps/plugins/export/exportgedcom.py [hunk 1]
```diff
@@ -513,14 +513,16 @@ def _gender(self, person):
         Write out the gender of the person to the file.
 
         If the gender is not male or female, simply do not output anything.
-        The only valid values are M (male) or F (female). So if the geneder is
-        unknown, we output nothing.
+        The only valid values are M (male), F (female) and X (other). So if
```
Comment (cdhorn) [inline gramps/plugins/export/exportgedcom.py:516]: X is only valid for Gedcom >= 7 so should this be handled differently here?

Comment (Nick-Hall) [reply gramps/plugins/export/exportgedcom.py:516]: Possibly.  I'm not sure which version of Gedcom we want to support in the export. We could just export "X" as "U".

Comment (cdhorn) [reply gramps/plugins/export/exportgedcom.py:516]: I think that might make sense for 5.5.1 and attach an auto generated FACT SEX attribute with the Other setting so at least the data doesn't get lost?

**Comment by cdhorn** [comment]
It is worth reviewing Tamura's recent assessment:

https://www.tamurajones.net/GenderInGenealogySoftware.xhtml

**Comment by Nick-Hall** [comment]
This is failing the unit test.  I need to investigate.

**Comment by liquidsunshine** [comment]
@Nick-Hall - I proposed [a way forward](https://gramps.discourse.group/t/additional-gender-options/2321/97) of handling the data around gender and sex in the forum thread.  I'm happy to help with development and/or testing if that would be helpful.

**Comment by Nick-Hall** [comment]
@liquidsunshine Thanks for your offer to help.

This PR adds an existing gender option of "Other".  Using "Non-binary" may be better, but I was trying to avoid any term which may offend people.  By "Other" I mean someone whose gender identity cannot be described as either "Male" or "Female".

Recording sex and gender information from sources is probably best done using attributes.  We are currently discussing adding dates to attributes in PR #1427.

As far as I am concerned this PR is ready to be tested and merged.

**Comment by emyoulation** [comment]
For future cross-reference (no action needed)
See the following Discourse topic (thread) for beta testing discussion of "Other" gender aware features. The 1st posting also has links to related topics.  
https://gramps.discourse.group/t/add-missing-awareness-of-other-gender/6034

**Pull request merged**